hugegraph-server/hugegraph-dist/src/assembly/travis/install-rocksdb.sh

```
@@ -0,0 +1,135 @@
+    fi
+
+    if ! mkdir -p "$resource_target"; then
+        echo "Error: Cannot create target directory '$resource_target'." >&2
+        return 1
+    fi
+
+    cp -f "$dest_dir"/*.html "$dest_dir"/*.css "$resource_target" 2>/dev/null || true
+}
+
+function preload_toplingdb() {
+  local jar_file=$(find $LIB -name "rocksdbjni*.jar")
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)